src/teleop/gui/src/components/SAArmControls.vue

```diff
@@ -69,6 +75,25 @@ export default {
   methods: {
     updateArmEnabled: function (enabled) {
       this.arm_enabled = enabled;
+      if (enabled) {
+        this.sa_mode_service.callService(
+          new ROSLIB.ServiceRequest({ mode: "sa_enabled" }),
+          (result) => {
+            if (!result.success) {
+              alert("Failed to enable SA arm");
+            }
+          }
+        );
+      } else {
+        this.sa_mode_service.callService(
+          new ROSLIB.ServiceRequest({ mode: "sa_disabled" }),
+          (result) => {
+            if (!result.success) {
+              alert("Failed to disable SA arm");
```

Comment:
Could we just make this mirror the RA side where the mode is stored as a string so we don't have to do this?